config/routes.rb

```diff
@@ -37,5 +37,8 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
 
+  get "/applications/new", to: "applications#new"
+  post "/applications/:id", to: "applications#create"
```

Comment:
here as well should just be `post "/applications", to: "applications#create"` because the create is redirecting.

Reply:
Thank you for pointing this out - now updated!